database/migrations/1628079946701_users.ts

```diff
@@ -5,17 +5,17 @@ export default class UsersSchema extends BaseSchema {
 
     public async up() {
         this.schema.createTable(this.tableName, (table) => {
```

Comment:
Ah you need to do migrations on top of the model I mentioned earlier?
There is not thight correlation there?

In other words there can become a drift?

Reply:
I am not sure if I understand your question, however, when creating models and tables together using **node ace make:model Order -m** it ideally creates an orders table and Order model. So basically migrations are for manipulating the table structure and models are for manipulating table data, that is why an Orders model maps to the orders table under the hood in Adonis

Reply:
Drift is when there are two seperate places that really depend on the same thing, the DB. As i commented here: https://github.com/arnoldkunihira/pizzeria-api/pull/4#discussion_r684825342

The model dont work unless the migraiton is right, but the migration really depends on the model then and preferebly not the other way around

Reply:
This is very clear and the way NestJs works which use typeORM and Adonis is kind of similar since it is all ORMs but differs in implementation.
In adonis, if the model is different from the migration then it will throw an exception and pinpoint where the problem is so that you fix it.